composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "woocommerce/action-scheduler-job-framework",
     "description": "A job framework for Action Scheduler (actionscheduler.org)",
-    "version": "1.0.0",
+    "version": "1.1.0",
```

Comment:
Semantic? WordPress versioning? Might as well decide on a convention now :)

Reply:
I feel like for "not-plugins" projects semantic versioning is the norm. Are you suggesting it should be 2.0.0 to be semantic?

Reply:
Not making any suggestion - happy with whatever you decide. I agree, 1.1 is semantic and signals the breaking change, and semantic works for this project IMO :)

So next release would be 1.1.1 right - unless it had breaking changes.

Reply:
Technically it should be 2.0 due to the breaking change. Minor versions are supposed to be fully backwards compatible...

https://semver.org/

Reply:
Interesting! Well my only feedback is be explicit about what the versioning policy is.

Reply:
Sounds good, I've updated the readme to state it uses Semver and bumped this version to 2.0.0.

Reply:
💥